pakyow-core/spec/unit/middleware/req_path_normalizer_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'spec_helper'
+require 'core/middleware/req_path_normalizer'
+
+describe Pakyow::Middleware::ReqPathNormalizer do
+  let :app do
+    double
+  end
+
+  let :instance do
+    Pakyow::Middleware::ReqPathNormalizer.new(app)
+  end
+
+  let :env do
+    {
+      'PATH_INFO' => path,
+      'REQUEST_METHOD' => method
+    }
+  end
+
+  let :path do
+    '//test_page'
+  end
+
+  let :normalized_path do
+    instance.normalize_path(env['PATH_INFO'])
+  end
+
+  let :method do
+    'GET'
+  end
+
+  before do
+    allow(Pakyow).to receive(:app).and_return(app)
+    allow(app).to receive(:call)
+  end
+
+  describe '#call' do
+    it 'exists' do
+      expect(instance).to respond_to(:call)
+    end
+
+    it 'accepts one arg' do
+      expect(instance.method(:call).arity).to eq(1)
+    end
+
+    context 'path contains //' do
+      it 'issues a 301 redirect with the normalized path' do
+        expect_any_instance_of(Pakyow::CallContext).to receive(:redirect).with(normalized_path, 301)
```

Comment:
I added all the required files but I'm still getting this
```
Failure/Error: path, format = String.split_at_last_dot(path)

     NoMethodError:
       undefined method `split_at_last_dot' for String:Class
     # ./lib/core/request.rb:112:in `set_request_format_from_path'
     # ./lib/core/request.rb:82:in `setup'
     # ./lib/core/request.rb:21:in `initialize'
     # ./lib/core/call_context.rb:21:in `new'
     # ./lib/core/call_context.rb:21:in `initialize'
     # ./lib/core/middleware/req_path_normalizer.rb:16:in `new'
     # ./lib/core/middleware/req_path_normalizer.rb:16:in `call'
     # ./spec/unit/middleware/req_path_normalizer_spec.rb:49:in `block (4 levels) in <top (required)>'
```
Should I require the sting class that is in `lib/support`?